Sources/AccountSDKIOSWeb/Client/Client.swift

```diff
@@ -168,14 +186,19 @@ public class Client: CustomStringConvertible {
             preconditionFailure("Couldn't create loginURL")
         }
         
+        tracker?.interaction(.view, with: .webBrowser, additionalFields: [.getLoginSession(withMFA), .loginHint(loginHint), .withAssertion((assertion != nil) ? true : false), .extraScopeValues(extraScopeValues)])
```

Comment:
nit: very long line, can we break it up?

Reply:
Done